<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeaponArrangementWidget</class>
 <widget class="QWidget" name="WeaponArrangementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>624</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mainHandThac0Group">
     <property name="title">
      <string>THAC0</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="proficiencyThac0ModifierLabel">
        <property name="text">
         <string>Proficency modifier</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="proficiencyThac0Modifier">
        <property name="minimum">
         <number>-5</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="styleModifierLabel">
        <property name="text">
         <string>Style modifier</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="styleModifier">
        <property name="minimum">
         <number>-4</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="weaponThac0ModifierLabel">
        <property name="text">
         <string>Weapon modifer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="weaponThac0Modifier">
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mainHandDamageGroup">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>410</height>
      </size>
     </property>
     <property name="title">
      <string>Damage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QToolBox" name="mainHandDamage">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="phisicalDamage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>211</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Physical</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="proficiencyDamageModifierLabel">
              <property name="text">
               <string>Proficency modifier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="proficiencyDamageModifier">
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="damageDiceLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="weaponDamageDiceLabel">
              <property name="text">
               <string>Weapon</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="weaponDamageDiceSide">
              <property name="suffix">
               <string> sides</string>
              </property>
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
              <property name="value">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="weaponDamageDiceBonus">
              <property name="suffix">
               <string> bonus</string>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="weaponDamageDiceNumber">
              <property name="suffix">
               <string> dice</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="damageType1Label">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="damageType"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="damageDetail">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="SpecialDamageWidget" name="acidDamage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>211</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Acid</string>
         </attribute>
        </widget>
        <widget class="SpecialDamageWidget" name="coldDamage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>211</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Cold</string>
         </attribute>
        </widget>
        <widget class="SpecialDamageWidget" name="electricityDamage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>211</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Electricity</string>
         </attribute>
        </widget>
        <widget class="SpecialDamageWidget" name="fireDamage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>176</width>
           <height>211</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Fire</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="attacksPerRoundLayout">
     <item>
      <widget class="QLabel" name="attacksPerRoundLabel">
       <property name="text">
        <string>Attacks Per Round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="attacksPerRound1">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="attacksPerRound2">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="criticalHitChanceLabel">
       <property name="text">
        <string>Critical hit chance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="criticalHitChance">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpecialDamageWidget</class>
   <extends>QWidget</extends>
   <header>damagecalculatorpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
